### PR TITLE
[Lib] Add epel release repo for rhel8

### PR DESCRIPTION
**Pull request by srivickynesh**
Signed-off-by: Sri Vignesh <vigneshsri11@gmail.com>

### CODE REVIEW

File: glustolibs-gluster/glustolibs/gluster/lib_utils.py [hunk 1]
```diff
@@ -697,6 +697,14 @@ def install_epel(servers):
                 if ret != 0:
                     g.log.error("Epel install failed")
                     rt = False
+            elif (("release 8" in release_string) or
```
Comment (pranavprakash20) [inline glustolibs-gluster/glustolibs/gluster/lib_utils.py:700]: @srivickynesh Just to understand, lets say if its 'Fedora' in the release string, the control goes to the elif condition above (https://github.com/gluster/glusto-tests/blob/d147a419e5c99484a51cfcb956356748c45d76ab/glustolibs-gluster/glustolibs/gluster/lib_utils.py#L693) and not this elif statement. Is that acceptable or do we have to change the logic here?

**Review by pranavprakash20 — CHANGES_REQUESTED** [review]

**Review by pranavprakash20 — APPROVED** [review]
LGTM

File: glustolibs-gluster/glustolibs/gluster/lib_utils.py [hunk 2]
```diff
@@ -689,14 +689,21 @@ def install_epel(servers):
...
                                   "pub/epel/epel-release-latest-7.noarch.rpm")
                 if ret != 0:
                     g.log.error("Epel install failed")
                     rt = False
+            elif (("release 8" in release_string) or
+                  ("Fedora" in release_string)):
+                ret, _, _ = g.run(server,
+                                  "yum -y install http://dl.fedoraproject.org/"
+                                  "pub/epel/epel-release-latest-8.noarch.rpm")
+                if ret != 0:
```
Comment (kshithijiyer) [inline glustolibs-gluster/glustolibs/gluster/lib_utils.py:704]: Shouldn't this be `if ret`?

**Review by kshithijiyer — CHANGES_REQUESTED** [review]
Minor fix in if

**Review by kshithijiyer — APPROVED** [review]
LGTM

**Review by balakreddy — APPROVED** [review]

**Pull request merged**